modules/nf-core/orthofinder/tests/main.nf.test

```diff
@@ -52,7 +51,51 @@ nextflow_process {
             def stable_files = all_files.findAll { it.name in stable_file_names }
 
             assert snapshot(
-                all_file_names,
+                stable_files,
```

Comment:
To avoid test fails, `stable_files` should be swapped with `stable_files.toSorted()`.

Reply:
Great, I see now